AndelaOSP/Andela-Socials
# chore(events): update displayed message for no events

#### What Does This PR Do?
This PR updates the feedback message on EventsPage when there are no upcoming events.

#### Description Of Task To Be Completed
- rename EventNotFound component to NoEvents
- rename and update scss file
- update EventsPage component

#### Any Background Context You Want To Provide?
N/A

#### How can this be manually tested?
- pull branch
- navigate to `/events`. Appropriate message should be displayed if there are no events.

#### What are the relevant pivotal tracker stories?
[#166284452](https://www.pivotaltracker.com/story/show/166284452)

#### Screenshot(s)
